pkg/config/config.go

```diff
@@ -65,9 +65,11 @@ type Kafka struct {
 }
 
 type Google struct {
-	ClientSecret string
-	ClientID     string
-	AppName      string
+	ClientSecret  string
+	ClientID      string
+	AppName       string
+	GCSBucketName string
```

Comment:
How do we classify which image in the bucket used for avatar image or document files?